packages/terra-framework-docs/src/terra-dev-site/doc/table/About.1.doc.mdx

```diff
@@ -8,19 +8,53 @@ import TablePropsTable from "terra-table/lib/Table?dev-site-props-table";
+  * [Single Row Selection](/components/cerner-terra-framework-docs/table/examples/table-single-row-selection)
+  * [Multiple Row Selection](/components/cerner-terra-framework-docs/table/examples/table-multiple-row-selection)
+  * [Headerless Table](/components/cerner-terra-framework-docs/table/examples/table-without-headers)
+  * [Column Sorting](/components/cerner-terra-framework-docs/table/examples/sortable-table)
+  * [Column Error Indicator](/components/cerner-terra-framework-docs/table/examples/table-column-states)
+
+## Examples
+|Link to Example| Description|
+|-|-|
+|[Default Table](/components/cerner-terra-framework-docs/table/examples/default-table)|An example demonstrating a basic table populated with tabular data and no interactive elements.|
+|[Zebra Striped Table](/components/cerner-terra-framework-docs/table/examples/zebra-striped-table)|An example demonstrating the ability enable zebra striping for table rows.|
+|[Pinned Columns](/components/cerner-terra-framework-docs/table/examples/pinned-columns)|An example demonstrating pinned columns (frozen leftmost columns) and overflow columns.|
+|[Sortable Table](/components/cerner-terra-framework-docs/table/examples/sortable-table)|An example demonstrating how to enable sorting within a table. Heading cells of sortable columns are interactive and include an icon indicating sort direction.|
+|[Table Column States](/components/cerner-terra-framework-docs/table/examples/table-column-states)|An example demonstrating column states and interactions such as an error or sort. Header cells include an icon that represents the state of the data in the column.|
+|[Table with Focusable Cells](/components/cerner-terra-framework-docs/table/examples/table-focusable-cell)|An example demonstrating tab stop behavior in a table with interactable elements. Each tabbable element within a table will be a separate tab stop.|
+|[Single Row Selection](/components/cerner-terra-framework-docs/table/examples/table-single-row-selection)|An example demonstrating a table with single row selection mode. Consumers will control the selection state by leveraging the **onRowSelect** callback and **isSelected** row property.|
+|[Multiple Row Selection](/components/cerner-terra-framework-docs/table/examples/table-multiple-row-selection)|An example demonstrating a table with multiple row selection mode. Consumers will control the selection state by leveraging the **onRowSelect** callback and **isSelected** row property.|
+|[Table with Sections](/components/cerner-terra-framework-docs/table/examples/table-with-sections)|An example demonstrating how to create a table with sections.|
+|[Table Without Headers](/components/cerner-terra-framework-docs/table/examples/table-without-headers)|An example demonstrating how to hide table column headers.  The column headers will still exist in the DOM for a11y.|
+
 ## Accessibility
 ### Keyboard Interactions
```

Comment:
Should we be having keyboard interactions for the multiple row selection mode?

Reply:
There are no keyboard interactions for multiple row selection mode outside the ones specified.

Reply:
That is called out in the example for multiple row selection.  You are giving focus to the checkbox and that selects the row.  It would be a little difficult to differentiate in the About page with multiple modes.